lib/rototiller/task/rototiller_task.rb

```diff
@@ -33,19 +35,23 @@ def self.define_task(*args, &task_block)
         self.new(*args, &task_block)
       end
 
-      def add_env(env_vars)
-        (@env_vars << env_vars).flatten!
+      def add_env(*env_vars)
```

Comment:
we discussed two args signatures a user should be able to employ to add envs:
* add_env(env_nam_str, default_value_str, message_str)  # i forget all the args here.  beginnner mode
* add_env([EnvVar]) # as array or single EnvVar type